lib/mozart_fetcher/application.ex

```diff
@@ -32,7 +32,7 @@ defmodule MozartFetcher.Application do
        [
          name: :fetcher_cache,
          ttl_check_interval: :timer.seconds(1),
-         global_ttl: :timer.seconds(30)
+         global_ttl: :timer.seconds(to_ttl(System.get_env("local_cache_ttl")))
```

Comment:
I wonder if we should pick either upper or lower case. Not for this PR, but I think we use a few different casing conventions.

Reply:
This will be seconds and not milliseconds. The default timeout was 3000, but I don't see that in the build repo.

Reply:
Correct, seconds not millisecond.

Also t seems Fetcher uses lower case across the board, the one exception is DEV_CERT_PEM for the cert path on local development